src/usdm-client.ts

```diff
@@ -432,6 +507,54 @@ export class USDMClient extends BaseRestClient {
     return this.getPrivate('fapi/v1/commissionRate', params);
   }
 
+  getDownloadIdForFuturesTransactionHistory(
+    params: GetDownloadIdFuturesTransactionParams,
+  ): Promise<DownloadIdFuturesTransaction> {
```

Comment:
For the naming pattern, I normally do ProductItemAnythingElse

So here it would be `FuturesTransactionDownloadId`. This way, interfaces from the same product always have the same prefix (Futures).

Reply:
Same here for the fn name. Looking at the docs, perhaps this is better? `getFuturesTransactionHistoryDownloadId` ? It's a mouthful either way, but it's consistent with the naming pattern of other methods.